### PR TITLE
Drop everything but awssqs

### PR DESCRIPTION
**Pull request by lberk**
Fixes knative/community#232

## Proposed Changes

- Drop everything but the awssqs bits
- Move eventing-contrib/awssqs to eventing-awssqs

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -55,23 +55,22 @@ follows.
 
 ## Installing Sources
 
-Once you've [setup your development environment](#getting-started), install any
-of the sources _Github Source_, _AWS SQS Source_, _Camel Source_, _Kafka Source_
-with:
+Once you've [setup your development environment](#getting-started), install
+the _AWS SQS Source_ with:
```
Comment (mattmoor) [inline DEVELOPMENT.md:59]: Format markdown:
```suggestion
Once you've [setup your development environment](#getting-started), install the
_AWS SQS Source_ with:
```

File: README.md [hunk 2]
```diff
@@ -1,25 +1,14 @@
...
+[![go.dev reference](https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white)](https://pkg.go.dev/github.com/knative-sandbox/eventing-awssqs)
+[![Go Report Card](https://goreportcard.com/badge/knative-sandbox/eventing-awssqs)](https://goreportcard.com/report/knative/eventing-)
+[![Releases](https://img.shields.io/github/release-pre/knative-sandbox/eventing-awssqs.svg)](https://github.com/knative-sandbox/eventing-awssqs/releases)
+[![LICENSE](https://img.shields.io/github/license/knative-sandbox/eventing-awssqs.svg)](https://github.com/knative-sandbox/eventing-awssqs/blob/master/LICENSE)
 [![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://knative.slack.com)
 
-The Knative Eventing Contrib project provides source and channel implementations
-that:
+The Knative Eventing AWS SQS project provides a source implementation
+that registers for events on the specified SQS Queue
```
Comment (mattmoor) [inline README.md:11]: Format markdown:
```suggestion
The Knative Eventing AWS SQS project provides a source implementation that
registers for events on the specified SQS Queue
```

File: samples/README.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
+# AWS SQS - Source
+
+These samples demonstrate how to configure an AWS SQS source, consuming events
+from an AWS SQS queue.
+
+## Deployment Steps
+
+The steps below assume that you have cloned the
+[eventing-awssqs](https://github.com/knative-sandbox/eventing-awssqs) repo and run the
+commands from the root.
```
Comment (mattmoor) [inline samples/README.md:10]: Format markdown:
```suggestion
[eventing-awssqs](https://github.com/knative-sandbox/eventing-awssqs) repo and
run the commands from the root.
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `prettier --write --prose-wrap=always $(find -name '*.md' | grep -v vendor | grep -v .github | grep -v docs/cmd/)`

File: DEVELOPMENT.md (hunk 1 above)
Comment (mattmoor) [inline DEVELOPMENT.md:59]: Format markdown:
```suggestion
Once you've [setup your development environment](#getting-started), install the
_AWS SQS Source_ with:
```

File: README.md (hunk 2 above)
Comment (mattmoor) [inline README.md:11]: Format markdown:
```suggestion
The Knative Eventing AWS SQS project provides a source implementation that
registers for events on the specified SQS Queue
```

File: samples/README.md (hunk 3 above)
Comment (mattmoor) [inline samples/README.md:10]: Format markdown:
```suggestion
[eventing-awssqs](https://github.com/knative-sandbox/eventing-awssqs) repo and
run the commands from the root.
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `prettier --write --prose-wrap=always $(find -name '*.md' | grep -v vendor | grep -v .github | grep -v docs/cmd/)`

File: README.md [hunk 4]
```diff
@@ -1,25 +1,14 @@
...
-- Integrate with Apache CouchDB
-- Integrate with Apache Kafka
-- Integrate with AWS SQS
-- Integrate with Ceph
-- Integrate with GitHub
-- Integrate with GitLab
-- Integrate with NATS Streaming
-- Integrate with Prometheus
-- Integrate with Websockets
-- Expose an ingress
```
Comment (mattmoor) [inline README.md:12]: Format markdown:
```suggestion

```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `prettier --write --prose-wrap=always $(find -name '*.md' | grep -v vendor | grep -v .github | grep -v docs/cmd/)`

**Comment by lionelvillard** [comment]
/lgtm
Thanks!

**Pull request merged**